### ISSUE TITLE
Error: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1 

### ISSUE THREAD

**Issue by kongakong** [alexharris/octopress-flat#3]
After installing the theme, I ran 'rake generate' and got this error message: 

```
$ rake generate
## Generating Site with Jekyll
unchanged sass/screen.scss
Configuration from /Users/antkong/octopress/_config.yml
Building site: source -> public

ERROR: YOUR SITE COULD NOT BE BUILT:
------------------------------------
(<unknown>): did not find expected key while parsing a block mapping at line 2 column 1 in /path/to/markdown.file
```

Here is the content of the mark down file

```

---
layout: post
title: "Finally finished the book, “Winston’s War: Churchill, 1940-1945” by Max Hastings"
date: 2012-03-13 22:00
comments: true
categories: Winston Churchill

---

Finally finished the book, “Winston’s War: Churchill, 1940-1945” by Max Hastings


```

At first I hoped this may be [the cause of the error](http://conghui.github.io/2013/04/27/did-not-find-expected-key-while-parsing-a-block-mapping/). But after removing some double quote/single quote, I still got the same error message.

**Comment by alexharris** [comment]
Hi,

So just to confirm, you did try creating a new post with no quotation marks in the title? Maybe try with a very simple title like: 

``` markdown
title: "Test"
```

Just to confirm.

Also, can you try switching to a different theme, and try creating the same post with that theme? That will help us to determine if it is indeed a problem with the theme, or with the post itself.

thanks,
Alex

**Comment by kongakong** [comment]
I had switched back to the classic theme and it can render fine.

As suggested, I created a test post, switched to octopress-flat and rerun 'rake generate'. This time around the 'rake generate' error message is gone.

I think it has something to do with a tag cloud plugin. After switching the themes I have lost my theme customisation which added a tag cloud page to my blog. 

I suppose the issue is resolved. I will close it later on.

Thanks for looking into it.
